pkg/cmd/server/server.go

```diff
@@ -100,6 +110,10 @@ const (
 	defaultBackupTTL = 30 * 24 * time.Hour
 )
 
+var (
+	scheme = runtime.NewScheme()
```

Comment:
Can you add some comments to explain why this and the init function are doing scheme calls? It's not clear to me why this is happening in the server.go file.